cabal.project

```diff
@@ -43,4 +43,23 @@ optional-packages:
   ./clash-cores
 
 allow-newer:
-  brick:base
+  brick:base,
+  vector-binary-instances:base,
+  cryptohash-sha256:base
+
+source-repository-package
+  type: git
+  location: https://github.com/kcsongor/generic-lens.git
+  tag: 8e1fc7dcf444332c474fca17110d4bc554db08c8
+  subdir: generic-lens-core
+
+source-repository-package
+  type: git
+  location: https://github.com/kcsongor/generic-lens.git
+  tag: 8e1fc7dcf444332c474fca17110d4bc554db08c8
+  subdir: generic-lens
```

Comment:
Why do we need these? I haven't needed these when I was testing on ghc9

Reply:
But you tested with the head overlay, right?

Reply:
```
[ 4 of 24] Compiling Data.Generics.Internal.Profunctor.Iso ( src/Data/Generics/Internal/Profunctor/Iso.hs, dist/build/Data/Generics/Internal/Profunctor/Iso.o, dist/build/Data/Generics/Internal/Profunctor/Iso.dyn_o )

src/Data/Generics/Internal/Profunctor/Iso.hs:76:7: error:
    • Couldn't match type: p0 i0 a b -> p0 i0 s t
                     with: forall (p :: * -> * -> * -> *) i.
                           Profunctor p =>
                           p i a b -> p i s t
      Expected: (s -> a) -> (b -> t) -> Iso s t a b
        Actual: (s -> a) -> (b -> t) -> p0 i0 a b -> p0 i0 s t
    • In the expression: dimap
      In an equation for ‘iso’: iso = dimap
    • Relevant bindings include
        iso :: (s -> a) -> (b -> t) -> Iso s t a b
          (bound at src/Data/Generics/Internal/Profunctor/Iso.hs:76:1)
   |
76 | iso = dimap
   |       ^^^^^
cabal: Failed to build generic-lens-core-2.1.0.0 (which is required by
```

Reply:
ah, right, I do test with the head overlay.